index.html

```diff
@@ -11,49 +11,62 @@
     <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <nav>
-      <h1 class="blogName"><img src ="images/logo.png" id="logo">
-        <strong>Delicious</strong><p class="subtitle">The best food blog on the web</p>
-        <section>
-          <img src="images/fb-icon.png" class="social">
-          <img src="images/twit-icon.png" class="social">
-          <img src="images/gp-icon.png" class="social">
-          <img src="images/insta-icon.png" class="social">
-          <img src="images/flic-icon.png" class="social">
-          <img src="images/pint-icon.png" class="social">
-          <img src="images/rss-icon.png" class="social">
-          <img src="images/mail-icon.png" class="social">
-        </section>
-      </h1>
-    </nav>
+    <header>
+      <nav>
+        <h1 class="blogName"><img src ="images/logo.png" id="logo">
```

Comment:
This h1 shouldn't wrap what it's wrapping. You're mainly going to have text in an h1 in most cases.